### PR TITLE
1.fix generation of uuid;

### PR DESCRIPTION
**Pull request by zlm0125**
Change-Id: Iff73ad95f87a81bc66ecc26e9ab250a8ca41ebf6

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible?

### CODE REVIEW

File: apisix/init.lua [hunk 1]
```diff
@@ -107,6 +107,10 @@ function _M.http_init_worker()
     require("apisix.debug").init_worker()
     require("apisix.upstream").init_worker()
 
+    -- need call seed
+    local uuid = require("resty.jit-uuid")
+    uuid.seed()
```
Comment (membphis) [inline apisix/init.lua:112]: https://github.com/apache/apisix/blob/master/apisix/init.lua#L72

We have set the seed with a better way. So we do not need to set it agian.

Comment (zlm0125) [reply apisix/init.lua:112]: But in k8s, it not work very well.

Comment (membphis) [reply apisix/init.lua:112]: you can print more log for confirming why it works badly. we need to know the reason.

Comment (moonming) [reply apisix/init.lua:112]: @ zlm0125  ping

**Comment by membphis** [comment]
resolve this bug by another PR: https://github.com/apache/apisix/pull/2357
I think we can close this PR. @zlm0125

**Pull request closed without merging**